common/src/main/java/com/wynntils/screens/guides/charm/WynntilsCharmGuideScreen.java

```diff
@@ -30,7 +31,9 @@ public final class WynntilsCharmGuideScreen
     private List<GuideCharmItemStack> allCharmItems = List.of();
 
     private WynntilsCharmGuideScreen() {
-        super(Component.translatable("screens.wynntils.wynntilsGuides.charmGuide.name"));
+        super(
+                Component.translatable("screens.wynntils.wynntilsGuides.charmGuide.name"),
+                List.of(ItemFilterType.GEAR, ItemFilterType.GEAR_INSTANCE));
```

Comment:
All guides should also use GENERIC

Reply:
Why? I think you don't need more than I listed.

Reply:
Generic includes level and favourite